SAMv1.tex

```diff
@@ -312,10 +312,10 @@ \subsection{The header section}
   & {\tt LB} & Library.\\\cline{2-3}
   & {\tt PG} & Programs used for processing the read group.\\\cline{2-3}
   & {\tt PI} & Predicted median insert size.\\\cline{2-3}
-  & {\tt PL} & Platform/technology used to produce the reads. \emph{Valid values}:
-  {\tt CAPILLARY}, {\tt LS454}, {\tt ILLUMINA}, {\tt SOLID}, {\tt HELICOS}, {\tt IONTORRENT}, {\tt ONT}, and {\tt PACBIO}.\\\cline{2-3}
-  & {\tt PM} & Platform model. Free-form text providing further details of the
-  platform/technology used.\\\cline{2-3}
+  & {\tt PL} & Platform/technology used to produce the reads. \emph{Valid values are}:
+ {\tt CAPILLARY},  {\tt DNB} (MGI/BGI's DNA NanoBalls), {\tt HELICOS}, {\tt ILLUMINA} (can be used for either 4- or 2-color chemistry, patterned or unpatterned flowcells), {\tt IONTORRENT}, {\tt LS454}, {\tt ONT} (Oxford Nanopore), {\tt PACBIO} (Pacific Biotechnology), {\tt SOLID}.
```

Comment:
"DNB" is very nonspecific.  MGI I think acquired this from Complete Genomics, but the instruments are very different.  It's more like a data prep thing, which we tend to not describe here.  Eg rolling circle amplification, mate pair vs paired-end sequencing, etc.   I think it's not a good term for the PLatform field.

I also dislike all the extra verbiage which IMO isn't necessary here.